tox.ini

```diff
@@ -22,7 +22,7 @@ deps =
 commands = pytest -sv \
     --cov=juju_verify \
     --cov-fail-under 100 \
-    --cov-report=term \
+    --cov-report=term-missing \
```

Comment:
What is difference here?

Reply:
with `term-missing` the output will also show line numbers that were missed. Without it, only a percentage of covered lines is displayed.